pkgs/development/python-modules/crc32c/default.nix

```diff
@@ -0,0 +1,24 @@
+{lib, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "crc21c";
+  version = "1.7";
+
+  src = fetchFromGitHub {
+    owner = "ICRAR";
+    repo = "crc32c";
+    rev = "v${version}";
+    sha256 = "0c3wwxwkm7ax6lw5w053iw16izaqd6fq6j5185fnv8mg6liflaad";
+  };
+
+  checkInputs = [ ];
```

Comment:
Any reason to keep these around if they are empty?

Reply:
@mmahut , right, removed.